src/components/Title.svelte

```diff
@@ -0,0 +1,15 @@
+<script>
```

Comment:
I wonder if maybe we should call this component `PageTitle`, since that's how it's used?